orchestrator-salt/src/main/resources/salt-common/salt/filecollector/init.sls

```diff
@@ -0,0 +1,88 @@
+{%- from 'telemetry/settings.sls' import telemetry with context %}
+{%- from 'filecollector/settings.sls' import filecollector with context %}
+{%- from 'fluent/settings.sls' import fluent with context %}
+
+install_fluent_logger:
+  cmd.run:
+    - name: pip install fluent-logger>=0.9.6 --ignore-installed
+    - unless: pip list --no-index | grep -E 'fluent-logger'
+
+install_yaml:
+  cmd.run:
+    - name: pip install PyYAML --ignore-installed
+    - unless: pip list --no-index | grep -E 'PyYAML'
+
+install_pid:
+  cmd.run:
+    - name: pip install pid --ignore-installed
+    - unless: pip list --no-index | grep -E 'pid'
+
+/opt/filecollector:
+  file.directory:
+    - name: /opt/filecollector
+    - user: "root"
+    - group: "root"
+    - mode: 740
```

Comment:
There are lots of files being managed and have 740 and 750 mode. Do any of these files contain sensitive data or should we use 700 and 600 regardless? Why would a regular user need to read these files?

Reply:
yes it should be 640, will fix that in a fix commit, but I will wait until Lajos's review, in oder to fix his opened issues together with this